src/actions/index.js

```diff
@@ -10,9 +10,17 @@ export const DECREMENT = 'DECREMENT';
 // is left to the reducer(s).
 
 export const increment = () => {
+  console.log('incrementing')
```

Comment:
I find it useful to make my console logs all uppercase or like `!!! INCREMENTING` to make them stand out more